README.adoc

```diff
@@ -170,6 +166,29 @@ config.servicemanager.http_proxy_password = <Proxy user password>
 
 When these settings are applied, they are passed through to the Docker and OpenShift services. In an unauthenticated proxy environment, the `http_proxy_user` and `http_proxy_password` configurations can be omitted.
 
+=== Configuring Services
+The vagrant-service-manager helps you configure the service of your choice:
+
+. Enable the desired service(s) in the ADB Vagrantfile as:
++
+`config.servicemanager.services = 'openshift'`
++
+[NOTE]
+====
+- Docker is the default service for the Atomic Developer Bundle and does not require any configuration to ensure it is started whereas, the Red Hat Enterprise Linux Container Development Kit, which is based on ADB, automatically starts OpenShift. +
+- You can use a comma-separated list to enable multiple services. For instance: docker, openshift.
+====
+
+. Enable the relevant option for the services you have selected in the Vagrantfile. For example, specific versions of OpenShift can be set using the following variables:
++
+
+- `config.servicemanager.openshift_docker_registry = "docker.io"` - Specifies the registry from where the service should be pulled.
```

Comment:
This is not related to OpenShift afaik. This is really a Docker service configuration option. To use a custom OpenShift version you need 'openshift_image_name' and 'openshift_image_tag'. @budhrg, correct?

@Preeticp Any chance you actually tested this? Is this still working end to end?